assets/setup-wizard/welcome/index.js

```diff
@@ -1,92 +1,83 @@
 /**
  * WordPress dependencies
  */
+import { applyFilters } from '@wordpress/hooks';
 import { __ } from '@wordpress/i18n';
-import { Button, Card, CardBody } from '@wordpress/components';
-import { useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { UsageModal } from './usage-modal';
 import { useQueryStringRouter } from '../../shared/query-string-router';
 import { useSetupWizardStep } from '../data/use-setup-wizard-step';
 import { H } from '../../shared/components/section';
 
 /**
  * Welcome step for Setup Wizard.
  */
-export const Welcome = () => {
-	const [ usageModalActive, toggleUsageModal ] = useState( false );
-
+const Welcome = () => {
 	const { goTo } = useQueryStringRouter();
 
-	const {
-		stepData,
-		submitStep,
-		isSubmitting,
-		errorNotice,
-	} = useSetupWizardStep( 'welcome' );
+	const { submitStep, isSubmitting, errorNotice } = useSetupWizardStep(
+		'welcome'
+	);
 
 	const onSubmitSuccess = () => {
-		toggleUsageModal( false );
 		goTo( 'purpose' );
 	};
 
-	const submitPage = ( allowUsageTracking ) => {
-		submitStep(
-			{ usage_tracking: allowUsageTracking },
-			{ onSuccess: onSubmitSuccess }
-		);
+	const submitPage = () => {
+		submitStep( {}, { onSuccess: onSubmitSuccess } );
 	};
 
+	/**
+	 * Filters the title from the Welcome step in the Setup Wizard.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param {string} title Title text.
+	 *
+	 * @return {string} Filtered title text.
+	 */
+	const title = applyFilters(
```

Comment:
The `title` and `paragraph` have a filter, so we can customize this text when using Sensei Pro.